actors/builtin/miner/policy.go

```diff
@@ -190,21 +245,28 @@ type VestSpec struct {
 	Quantization abi.ChainEpoch // Maximum precision of vesting table (limits cardinality of table).
 }
 
+// PARAM_SPEC
 var PledgeVestingSpec = VestSpec{
 	InitialDelay: abi.ChainEpoch(180 * builtin.EpochsInDay), // PARAM_FINISH
 	VestPeriod:   abi.ChainEpoch(180 * builtin.EpochsInDay), // PARAM_FINISH
 	StepDuration: abi.ChainEpoch(1 * builtin.EpochsInDay),   // PARAM_FINISH
 	Quantization: 12 * builtin.EpochsInHour,                 // PARAM_FINISH
 }
```

Comment:
**Clarification request:** hey @zixuanzh, can you clarify the following?

Question:
- Miner pledges 20BR to add a sector
- Assume for a second that the miner has never mined any block (hence has no reward)
- After 180 days, the miner will get some of the pledge back (vesting period starts)
- Will the miner on day 181 have less than 20BR? In my understanding a pledge of 20BR must be constant for a miner sector to be active.

What I want to avoid:
- The pledge for a small miner that is +180 days in won't be enough to cover a minimum consensus fault fee
- The pledge is not enough to cover FF/SPs

Reply:
We will be updating the policy, miners need to maintain the sum of Initial Pledges for all their active sectors.

Reply:
.. meaning that it won't vest.

Note that the current constructions can make no guarantee about being able to cover FF/SPs because they are calculated using a different BR as the base.